default.nix

```
@@ -38,7 +38,7 @@ let
 in
 buildGoApplication rec {
   pname = "chain-maind";
-  version = "4.2.8";
+  version = "sdk47-upgrade";
```

```suggestion
  version = "4.3.0";
```
the next version should be `4.3.0` by default.